File: VB/Q296004/Program.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Xpo.DB
Imports System.Windows.Forms

Namespace Q296004
	Friend NotInheritable Class Program
		''' <summary>
		''' The main entry point for the application.
		''' </summary>
		Private Sub New()
		End Sub
		<STAThread> _
		Shared Sub Main()
			Application.EnableVisualStyles()
			Application.SetCompatibleTextRenderingDefault(False)
			XpoDefault.DataLayer = XpoDefault.GetDataLayer(AccessConnectionProvider.GetConnectionString("..\..\Q296004.mdb"), AutoCreateOption.DatabaseAndSchema)
			Application.Run(New Form1())
		End Sub
	End Class
End Namespace

File: CS/Q296004/Program.cs
using System;
using DevExpress.Xpo;
using DevExpress.Xpo.DB;
using System.Windows.Forms;

namespace Q296004 {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main() {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            XpoDefault.DataLayer = XpoDefault.GetDataLayer(AccessConnectionProvider.GetConnectionString(@"..\..\Q296004.mdb"),
                AutoCreateOption.DatabaseAndSchema);
            Application.Run(new Form1());
        }
    }
}